introduction/de/04-benefits.asciidoc

```diff
@@ -0,0 +1,31 @@
+== Worin liegen die Vorteile von InnerSource?
+
+Die Zusammenarbeit mit Hilfe von InnerSource bietet viele Vorteile.
+InnerSource bietet einem Unternehmen eine skalierbare Strategie für *Gastteams, um Features zu dem Zeitpunkt zu erhalten, zu dem sie diese benötigen*, ohne langfristig auch die Wartung übernehmen zu müssen.
+Das Unternehmen als Ganzes gewinnt, da die Gastteams die so gewonnene Zeit anderen Aufgaben widmen können. 
+
+Während dieses Ergebnis ein glänzender Vorteil von InnerSource ist gibt es für Hosts die regelmäßige InnerSourcebeiträge erhalten ebenfalls viele Vorteile.
+Denken Sie daran, dass der Product Owner im Hostteam im Rahmen des InnerSourceprozesses von Anfang an entscheidet ob die bereitgestellten Funktionen gut und wünschenswert sind für das Project sind.
+Somit kann das Hostteam durch die Anwendung von InnerSourceprinzipien  *Hilfe bei der Erstellung eines besseren Produkts* für seine Verbraucher erhalten!
+
+*InnerSource bietet dem Hostteam eine skalierbare Strategie* zur Erfüllung unterschiedlicher Funktionsanforderungen die von seinen verschiedenen Nutzern/Kunden vorgebracht werden.
+Angesichts der festen Arbeitskapazität der Vollzeitmitglieder des Hostteams ist es wahrscheinlich, dass die kombinierten Geschäftspläne der Nutzer zuweilen sehr (oder sogar unangemessen) viel zusätzliche Arbeit für das Hostteam bedeutet.
+Ohne InnerSource führt diese Situation leicht zu einem gestressten, überlasteten Team, das sich mit vielen (fremden) Featureanfragen befasst die ggf. über seine Führungskräfte eskaliert worden sind.
+
+Wenn das Hostteam jedoch nach InnerSource Prinzipien arbeitet, werden die zum Erstellen dieser Funktionen erforderlichen technischen Lösungen in Form von Gastbeiträgen gemäss der Priorität des Gastteam erbracht.
```

Comment:
```suggestion
Wenn das Hostteam jedoch nach InnerSource Prinzipien arbeitet, werden die, zum Erstellen dieser Funktionen erforderlichen, technischen Lösungen in Form von Gastbeiträgen, gemäß der Priorität des Gastteam, erbracht.
```

Reply:
My German tells my that there's a missing Genitiv with _"Priorität des Gastteam**s**"_.
Also, while social scientist me appreciates that sentence, computer scientist me says it needs more full stops.

Reply:
fixed (per Johannes' comment) and accepted change suggestion